crates/recursor/src/recursor.rs

```diff
@@ -307,6 +307,10 @@ impl Recursor {
         request_time: Instant,
         query_has_dnssec_ok: bool,
     ) -> Result<Lookup, Error> {
+        if !query.name().is_fqdn() {
+            return Err(Error::from("Query's domain name must be fully qualified"));
```

Comment:
I think we usually avoid capitalization in error messages.

Reply:
updated the error message